services/src/test-library/src/main/java/org/openkilda/testing/ConstantsGrpc.java

```diff
@@ -0,0 +1,33 @@
+/* Copyright 2019 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.testing;
+
+public final class ConstantsGrpc {
+    public static final Integer SWITCH_PORT = 31;
+    public static final Integer LOGICAL_PORT = 1031;
+    public static final String REMOTE_LOG_IP = "1.1.11.111";
+    public static final Integer REMOTE_LOG_PORT = 10514;
+    // TODO(andriidovhan) get remote log ip from db and switch ip
+    public static final String SWITCH = "1.1.1.2"; // should be set a correct ip
+    public static final String DEFAULT_REMOTE_LOG_IP = "1.1.1.3"; // should be set a correct ip
+    public static final Integer DEFAULT_REMOTE_LOG_PORT = 514;
+    public static final String DEFAULT_LOG_MESSAGES_STATE = "off";
+    public static final String DEFAULT_LOG_OF_MESSAGES_STATE = "on";
```

Comment:
A lot of questions about all of these constants. Any of these cannot be assumed as constants.